go.mod

```diff
@@ -56,6 +56,8 @@ require (
 
 replace mellium.im/sasl => github.com/mellium/sasl v0.2.1
 
+replace github.com/fxamacker/cbor/v2 => github.com/turbolent/cbor/v2 v2.2.1-0.20200911003300-cac23af49154
```

Comment:
@turbolent I have to add this otherwise it doesn't build. Is this ok?

Reply:
Ah, right. The replace is only local to the package itself (i.e. onflow/cadence), and won't be applied in packages importing it (like here, flow-go). Let me look into this

Reply:
Asked the rest of the team here: https://axiomzen.slack.com/archives/CEXKUH74G/p1601594114204500